tests/functional/spec/list_collector_section_summary_items.spec.js

```diff
@@ -0,0 +1,133 @@
+import AnyCompaniesOrBranchesDrivingQuestionPage from "../generated_pages/list_collector_section_summary_items/any-companies-or-branches.page.js";
+import AnyCompaniesOrBranchesPage from "../generated_pages/list_collector_section_summary_items/any-other-companies-or-branches.page.js";
+import AnyCompaniesOrBranchesAddPage from "../generated_pages/list_collector_section_summary_items/any-other-companies-or-branches-add.page.js";
+import AnyCompaniesOrBranchesRemovePage from "../generated_pages/list_collector_section_summary_items/any-other-companies-or-branches-remove.page.js";
+import SectionSummaryPage from "../generated_pages/list_collector_section_summary_items/section-summary.page.js";
+
+describe("List Collector Section Summary Items", () => {
+  describe("Given I launch the test list collector section summary items survey", () => {
+    beforeEach(() => {
+      browser.openQuestionnaire("test_list_collector_section_summary_items.json");
+    });
+    it("When I answer yes to the driving question, Then I should be able to add my own item and the relevant data about it and get to the relevant page.", () => {
```

Comment:
This could be more specific?
```suggestion
    it("When I answer yes to the driving question, Then I should be able to add my own item and the relevant data about it and get to the list collector question page.", () => {
```